File: week02/week02-4.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        int H[256]={};//初始值都是0
        for(char c:s){
            H[c]++;//多一個字母C
        }
        for(char c:t){
            H[c]--;//用到一個字母c,陣列資料變少
            if(H[c]<0) return c;
        }
        return ' ';//最後都沒找到，就回傳空白字母
    }
};

File: week02/week02-4.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        d={}
        for c in s:
            if c in d:
                d[c]=d[c]+1
            else:
                d[c]=1
        

        for c in t:
            if c not in d:#不在字典裡,就是缺它，它就是多出來的
                return c
            if d[c]<=0:#不夠
                return c#ˊ找出多出來的字母
            else:
                d[c]=d[c]-1
        return